src/test/regress/expected/multi_metadata_sync.out

```diff
@@ -437,7 +437,7 @@ SELECT * FROM pg_dist_node ORDER BY nodeid;
 (4 rows)
 
 SELECT * FROM pg_dist_partition WHERE logicalrelid::text LIKE 'mx_testing_schema%' ORDER BY logicalrelid;
-          logicalrelid           | partmethod |                                                         partkey                                                          | colocationid | repmodel | autoconverted
+          logicalrelid           | partmethod |                                                        partkey                                                         | colocationid | repmodel | autoconverted
```

Comment:
Are those space changes expected?

Reply:
I'm not sure :D citus_indent does it and CI doesn't complain. So, should be good?